frontend/src/utils/getCurrentFiscal.ts

```diff
@@ -1,9 +1,8 @@
-import { FiscalKeys } from 'constants/fiscalKeys';
```

Comment:
I don't think we are using any of this fiscal/evaluation logic anymore right? can get rid of this too then.